bedrock/firefox/templates/firefox/welcome/page14.html

```diff
@@ -0,0 +1,82 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/welcome/base.html" %}
+
+{% block page_title %}{{ ftl('welcome-page10-page-title') }}{% endblock %}
+
+{% block page_image %}{{ static('img/products/vpn/common/social-share.png') }}{% endblock %}
+
+{% block body_class %}welcome-page14{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-picto') }}
+  {{ css_bundle('firefox_welcome_page10') }}
+{% endblock %}
+
+{% set _entrypoint = 'mozilla.org-firefox-welcome-14' %}
+{% set _utm_campaign = 'welcome-14-vpn' %}
+{% set source = '?source=welcome14' %}
+{% set show_coupon = switch('vpn-moments-coupon-code')%}
+
+{% block shoulder_cta %}{% endblock %}
+
+{% block content_intro %}
+  <section class="c-content-main">
+    <div class="mzp-l-content mzp-t-content-md">
+      <img class="c-main-image" src="{{ static('protocol/img/logos/mozilla/vpn/logo-flat.svg') }}" width="90" alt="">
+      <h1 class="c-main-title"> {% if show_coupon %}{{ ftl('welcome-page14-page-title') }}{% else %}{{ ftl('welcome-page14-page-title-alt') }}{% endif %}</h1>
+      <p class="c-main-tagline">{{ ftl('welcome-page10-main-description') }}</p>
+      {% if show_coupon %}
+        <div class="c-coupon-container">
+          <h3>{{ ftl('welcome-page14-use-code') }}</h3>
+        </div>
+      {% endif %}
+      <div class="c-main-cta">
+        <a href="{{ url('products.vpn.landing') }}{{ source }}#pricing" class="mzp-c-button mzp-t-product mzp-t-xl" data-cta-text="Get Mozilla VPN" data-cta-type="button">{{ ftl('welcome-page10-get-mozilla-vpn') }}</a>
```

Comment:
Issue (blocking): This should use `vpn_product_referral_link()` (instead of `{{ source }}`) to track subscription referrals: https://bedrock.readthedocs.io/en/latest/firefox-accounts.html#tracking-same-site-links-for-mozilla-vpn

Reply:
oh, duh! I just grabbed the contents of `page10` since it's the same design